README.md

```diff
@@ -127,7 +127,7 @@ Contains sample code for building linkerd plugins. More information:
 ## Testing
 
 ```bash
-docker run -v `pwd`:/root/linkerd-examples --entrypoint=/root/linkerd-examples/.circleci/ci.sh buoyantio/linkerd:0.9.1
+docker run -v `pwd`:/root/linkerd-examples --entrypoint=/bin/sh buoyantio/linkerd:1.0.0 -c "apk update; apk add curl; /root/linkerd-examples/.circleci/ci.sh"
```

Comment:
hmm, yeah, this passes locally for me -- i published new 1.0.0 images to dockerhub yesterday, switching us back to the old base images. maybe ci cached the previous build of the 1.0.0 image? i can investigate.